_includes/header.html

```diff
@@ -21,7 +21,7 @@
         <div class="trigger">
           {% for path in page_paths %}
             {% assign my_page = site.pages | where: "path", path | first %}
-            {% if my_page.title %}
+            {% if my_page.title and !my_page.exclude != true %}
```

Comment:
@dirkgomez Looking at the existing content, I would expect to see it as a new **h3** in the [Usage](https://github.com/jekyll/minima#usage) section